lite/Makefile.w32

```diff
@@ -1,10 +1,13 @@
 # Makefile for Win32 using Watcom compiler.
 #
 # wmake -f Makefile.w32
-# - builds libxmp.dll and its import lib (libxmp.lib)
+# - builds libxmp-lite.dll and its import lib (libxmp-lite.lib)
```

Comment:
Same here, instead of generating the normal and lite library with two different names it could be some kind of configuration in the same Makefile once all licensing issues are sorted out and the -lite package doesn't need to be published separately.

Reply:
That would be most ideal and it'd reduce maintenance cost.  But it'll have to be a subject of another patchset.